### PR TITLE
Socint 241 survey update fulfilments

### PR DESCRIPTION
**Pull request by PeterBochel**
Updating survey-update event to 0.5.0 version. 
See here for example: https://github.com/ONSdigital/ssdc-shared-events/blob/main/event_dictionary/0.5.0/examples/social/surveyUpdate.example.json

### CODE REVIEW

File: event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SurveyUpdate.java [hunk 1]
```diff
@@ -32,4 +33,10 @@ void setSampleDefinitionFromJson(JsonNode data) {
   void setMetadataFromJson(JsonNode data) {
     this.metadata = data.toString();
   }
+
+  private List<SurveyFulfilment> allowedPrintFulfilments;
```
Comment (rpcatling) [inline event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SurveyUpdate.java:37]: member variables would normally be placed above the methods

File: event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SurveyUpdate.java [hunk 2]
```diff
@@ -32,4 +33,10 @@ void setSampleDefinitionFromJson(JsonNode data) {
...
+
+  private List<SurveyFulfilment> allowedSmsFulfilments;
```
Comment (rpcatling) [inline event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SurveyUpdate.java:39]: is it possible to set these lists  in the `setSampleDefinitionFromJson(JsonNode data)` method above? That way we won't have to duplicate code in RHSvc and CCSvc to load these lists.

**Review by rpcatling — CHANGES_REQUESTED** [review]
just a trivial one

**Review by SteveScorfield — APPROVED** [review]
Looks fine

File: event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SurveyUpdate.java (hunk 1 above)
Comment (PeterBochel) [reply event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/SurveyUpdate.java]: Have changed order

**Review by rpcatling — APPROVED** [review]

**Pull request merged**